src/main/kotlin/project2/Train.kt

```diff
@@ -0,0 +1,22 @@
+package project2
+
+import java.util.*
+
+class Train(schedule:MutableList<Int>) {
+
+    private var delayed:Boolean = false
+    private var schedule:List<Int> = schedule
+
+    public fun setDelayed(d: Boolean){
```

Comment:
Methoden wir `setDelayed` und `getDelayed` sind in Ordnung, in Kotlin aber unüblich. Alternativ kannst du sie einfach entfernen und `delayed` so konfigurieren:

    public var delayed: Boolean = false

Darauf zugreifen kann man dann so:

    var train = Train()
    train.delayed = true